node/docs/genmd.js

```diff
@@ -102,6 +102,8 @@ writeLine();
 writeLine('The build generates a vsts-task-lib.d.ts file for use when compiling tasks');
 writeLine('In the example below, it is in a folder named definitions above the tasks lib');
 writeLine();
+writeLine('## [Release notes](releases.md)');
```

Comment:
I see. I forgot about the comments on the function itself. Mainly my intended goal is to start a release notes MD. That way task authors can better understand the implications of picking up a new version of the lib.